cheatSheet/README.md

```diff
@@ -23,7 +23,8 @@ Name <---\*--
 
 ## Pattern Syntax
 
-A pattern is a Name or a Process where processes are substituted with any combination of:  
+A pattern is a Name or a Process where some (or no) processes that make up components are
```

Comment:
I'm having trouble making this sentence both short and easy to understand ...

Reply:
I'd say omit the sentence entirely? We don't need to teach pattern matching here, just remind people who already know it of the syntax.

Reply:
Ah, good plan.

Reply:
What do you think about the examples?